shared/actions/notifications.js

```diff
@@ -47,7 +47,7 @@ function* _listenSaga(): Saga.SagaGenerator<any, any> {
   const setHandlers = (dispatch, getState) => {
     const listeners = ListenerCreator(dispatch, getState, NotifyPopup)
     Object.keys(listeners).forEach(key => {
-      engine().setIncomingHandler(key, listeners[key])
+      engine().setIncomingActionCreators(key, () => [listeners[key]])
```

Comment:
i think you want this to be
```
engine().setIncomingActionCreators(key, listeners[key]) 
```
so those methods get called w/ params they need. i think with the current code they'll get called and the params will be dispatch and getState and not whats passed by engine